homeassistant/components/schluter/climate.py

```diff
@@ -0,0 +1,169 @@
+            raise UpdateFailed(f"Error communicating with Schluter API: {err}")
+
+    coordinator = DataUpdateCoordinator(
+        hass,
+        _LOGGER,
+        name="schluter",
+        update_method=async_update_data,
+        update_interval=SCAN_INTERVAL,
+    )
+
+    await coordinator.async_refresh()
+
+    async_add_entities(
+        SchluterThermostat(coordinator, idx, temp_unit, api, session_id)
+        for idx, ent in enumerate(coordinator.data)
+    )
+
+
+class SchluterThermostat(ClimateDevice):
+    """Representation of a Schluter thermostat."""
+
+    def __init__(self, coordinator, idx, temp_unit, api, session_id):
+        """Initialize the thermostat."""
+        self._unit = temp_unit
+        self.coordinator = coordinator
+        self.idx = idx
+        self._api = api
+        self._session_id = session_id
+        self._support_flags = SUPPORT_TARGET_TEMPERATURE
+
+    @property
+    def should_poll(self):
+        """Return if platform should poll."""
+        return False
+
+    @property
+    def supported_features(self):
+        """Return the list of supported features."""
+        return self._support_flags
+
+    @property
+    def unique_id(self):
+        """Return unique ID for this device."""
+        return self.coordinator.data[self.idx].serial_number
+
+    @property
+    def device_info(self):
+        """Return information about the device."""
+        return {
+            "identifiers": {
+                (SCHLUTER_DOMAIN, self.coordinator.data[self.idx].serial_number)
+            },
+            "name": self.coordinator.data[self.idx].name,
+            "manufacturer": "Schluter",
+            "model": "Thermostat",
+            "sw_version": self.coordinator.data[self.idx].sw_version,
+        }
+
+    @property
+    def name(self):
+        """Return the name of the thermostat."""
+        return self.coordinator.data[self.idx].name
+
+    @property
+    def temperature_unit(self):
+        """Return the unit of measurement."""
+        return self._unit
+
+    @property
+    def current_temperature(self):
+        """Return the current temperature."""
+        return self.coordinator.data[self.idx].temperature
+
+    @property
+    def hvac_mode(self):
+        """Return current operation ie. heat, idle."""
+        return (
+            HVAC_MODE_HEAT
+            if self.coordinator.data[self.idx].is_heating
+            else HVAC_MODE_OFF
+        )
+
+    @property
+    def target_temperature(self):
+        """Return the temperature we try to reach."""
+        return self.coordinator.data[self.idx].set_point_temp
+
+    @property
+    def hvac_modes(self):
+        """List of available operation modes."""
+        return None
+
+    @property
+    def fan_mode(self):
+        """Return whether the fan is on."""
+        # No Fan available
+        return None
+
+    @property
+    def fan_modes(self):
+        """List of available fan modes."""
+        return None
```

Comment:
This entity does not have a fan in its supported features, this doesn't need to be implemented and can be removed.

Reply:
This has been removed 👍